src/index.js

```diff
@@ -1,47 +1,48 @@
+import '@fortawesome/fontawesome-free/js/fontawesome.js';
```

Comment:
- [ ] Everything works great but it seems you missed one of the project requirements. You should create a new JavaScript file for new functionalities. Please extract the new functionalities to a new JavaScript file as requested in the project description 
Please check [the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Create%20a%20new%20JavaScript%20file%20for%20the%20new%20functionality.)

Reply:
Hi @sinansevgi.. I actually ended up not using it, I went another way. So, I'm just going to remove it.

Reply:
Hi @Ntare22, Sorry for the misunderstanding. My comment is not about the fort awesome. I attached my comment to the index.js file and it automatically marked the first line of the file. What I was trying to say is, you should create a new javascript file for all of your CRUD functionalities and import that file to your index.js file.